lore/util.py

```diff
@@ -299,15 +299,16 @@ def calling_logger(height=1):
     # Rollbar
     try:
         with timer('rollbar init', logging.DEBUG):
-            import rollbar
-            rollbar.init(
-                os.environ.get("ROLLBAR_ACCESS_TOKEN", None),
-                allow_logging_basic_config=False,
-                environment=env.NAME,
-                enabled=(env.NAME != env.DEVELOPMENT),
-                handler='blocking',
-                locals={"enabled": True}
-            )
+            if not getattr(sys, 'lore_disable_rollbar', False):
```

Comment:
I would make this a check for whether or not rollbar is present, instead of having to set a global on sys.

Reply:
Do you mean to check whether `rollbar` is installed?